src/DataMigration/DataMigration/help/New-AzDataMigrationToSqlManagedInstance.md

```diff
@@ -81,7 +81,7 @@ Accept wildcard characters: False
 ```
 
 ### -AzureBlobContainerName
-Blob container name where backups are stored.
```

Comment:
The description is generated. The change will be lost if you regenerate the codes.